.rubocop.yml

```diff
@@ -200,3 +200,745 @@ Metrics/ParameterLists:
 
 Metrics/PerceivedComplexity:
   Max: 7
+
+# Additional todo list not generated by --autogen-config
+Metrics/LineLength:
```

Comment:
⚠️ this cop is already defined in line 192. The same for the ones below it

Reply:
I know that they are already defined above. I see the definition above as our target these lines down here is intermediate todo list. What do you suggest? Should we comment out the definitions above and add a comment like `# look below for current todo list`? I don't think we should include all the files in the list above, because it will be very difficult to read the general style guide.

Reply:
aha, that's a good idea. Yes, I would explain it as I'm sure I'm won't be the last one that thinks that otherwise.